Library/Homebrew/dependency.rb

```diff
@@ -55,7 +55,7 @@ def installed?(minimum_version: nil)
     return false unless formula
 
     if minimum_version.present?
-      formula.any_version_installed? && (formula.any_installed_version.version >= minimum_version)
+      formula.any_version_installed? && (formula.any_installed_version >= minimum_version)
```

Comment:
It doesn't look like this usually gets called with `minimum_version` set so it's not a big deal to change the argument type from `Version` to `PkgVersion`.